hm_pyhelper/hardware_definitions.py

```diff
@@ -18,6 +18,7 @@ def is_raspberry_pi() -> bool:
         'APPNAME': 'Indoor',
         'SPIBUS': 'spidev1.2',
         'KEY_STORAGE_BUS': '/dev/i2c-1',
+        'SWARM_KEY_URI': 'ecc://i2c-1:96?slot=0',
```

Comment:
Not ideal to have two hardcoded values for i2c. Should the swarm key URI be a method in pyhelper like?

```
def get_swarm_key_uri(key_storage_bus):
  if key_storage_bus == '/dev/i2c-1'
    return 'ecc://i2c-1:96?slot=0'
  ...
```

Reply:
We are going to sunset the `KEY_STORAGE_BUS` and use only `SWARM_KEY_URI` to get the ECC address. Keeping the `KEY_STORAGE_BUS` is only for a while. 
@marvinmarnold Do you see some use cases that make us to keep `KEY_STORAGE_KEY` for long term?

Reply:
The only use case I can see for having key storage bus separate would be if we were using the ECC chip for some other authentication. Which we aren't at present so probably not relevant right now.

Reply:
I had put SWARM_KEY_URI in ticket description to follow upstream and the format seemed superior. It fit right into any url parser yielding protocol, bus, adddress and slot even if we have other uses of ecc.